README.md

```diff
@@ -50,7 +50,7 @@ fn start_background_audio(asset_server: Res<AssetServer>, audio: Res<Audio>) {
 - [x] control playback rate
 - [ ] control pitch (no change in playback rate)
 - [x] control panning
-- [ ] get the current status of a track (time elapsed/left)?
+- [x] get the current status of a track (time elapsed/left)?
```

Comment:
```suggestion
- [x] get the current status and position of a track
```

Reply:
We could also link to the new example here.

Reply:
> We could also link to the new example here.

I'll do this.